instance/tests/models/test_logging.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Logger models & mixins - Tests
+"""
+
+# Imports #####################################################################
+
+import pytz
+from datetime import datetime, timedelta
+from mock import patch
+
+from instance.tests.base import TestCase
+from instance.tests.models.factories.instance import OpenEdXInstanceFactory
+from instance.tests.models.factories.server import OpenStackServerFactory
+
+
+# Tests #######################################################################
+
+# Factory boy doesn't properly support pylint+django
+#pylint: disable=no-member
+
+class LoggerInstanceMixinTestCase(TestCase):
+    """
+    Test cases for LoggerInstanceMixin
+    """
+    def test_log_text(self):
+        """
+        Check `log_text` output for combination of instance & server logs
+        """
+        instance = OpenEdXInstanceFactory()
+        server = OpenStackServerFactory(instance=instance)
+        now = datetime(2015, 8, 5, 18, 7, tzinfo=pytz.utc)
+        sec = timedelta(seconds=1)
+
+        with patch('django.utils.timezone.now') as mock_now:
```

Comment:
@e-kolpakov Ah, thanks! I was looking for that lib, which I couldn't remember the name of. Patching datetime directly isn't that much more work though, but it will make the test clearer to read.